newIDE/app/src/InstructionOrExpression/EnumerateExpressions.js

```diff
@@ -23,10 +25,10 @@ const getExtensionPrefix = (extension: gdPlatformExtension): string => {
   const allBehaviorsTypes = extension.getBehaviorsTypes();
 
   if (allObjectsTypes.size() > 0 || allBehaviorsTypes.size() > 0) {
-    return (
-      (extension.getName() === 'BuiltinObject'
-        ? 'Common expressions for all objects'
-        : extension.getFullName()) + GROUP_DELIMITER
+    return extension.getName() === 'BuiltinObject' ? (
+      <Trans>Common expressions for all objects</Trans>
```

Comment:
This fix is more complex, it will involve either passing the translated string as an argument, or passing down the translation function+macro, as you're supposed to return a string, not a React Node.